open-metadata-implementation/adapters/open-connectors/event-bus-connectors/open-metadata-topic-connectors/kafka-open-metadata-topic-connector/src/main/java/org/odpi/openmetadata/adapters/eventbus/topic/kafka/KafkaOpenMetadataEventConsumer.java

```diff
@@ -52,7 +43,10 @@ public class KafkaOpenMetadataEventConsumer implements Runnable
     private long nextMessageProcessingStatusCheckTime = System.currentTimeMillis();
     private long maxNextPollTimestampToAvoidConsumerTimeout = 0;
     private final long maxMsBetweenPolls;
-    
+
+    // Keep track of when an initial rebalance is done
+    private static boolean initialPartitionAssignment = true;
```

Comment:
I wonder why this is static, can't there be more than one Event consumer as it is a thread? It would be good to have a comment detailing your thinking on why it is safe to be a static.

Reply:
Agreed. Will update. thanks for spotting

Reply:
resolved